hackvote/app/controllers/events_controller.rb

```diff
@@ -0,0 +1,41 @@
+class EventsController < ApplicationController
+
+  NUMBER_OF_VOTES = 3
+
+  def new
+	  @event = Event.new
+	  @cur_events = Event.where( completed: false)
+	  @past_events = Event.where( completed: true)
```

Comment:
Huh, I thought it would be the other way around. Given it seems rails generates an sql query and the sql database gets to figure out what items to return. Would getting all of the events and then filtering in ruby scale well if the number of events got really big?